docs/product-installation-guide/src/main/asciidoc/clustering-ps-configure-domain-proc.adoc

```diff
@@ -97,5 +138,18 @@ The following dialects are supported:
 ** PostgreSQL: `org.hibernate.dialect.PostgreSQL82Dialect`
 ** PostgreSQL plus: `org.hibernate.dialect.PostgresPlusDialect`
 ** Sybase: `org.hibernate.dialect.SybaseASE157Dialect`
++
```

Comment:
Please remove Sybase for now as it is not supported - https://access.redhat.com/articles/3405381